worlds/seaofthieves/Items/ItemAdder.py

```diff
@@ -0,0 +1,64 @@
+from ..Configurations import SotOptionsDerived
+from BaseClasses import MultiWorld, Region, Entrance, ItemClassification
+from ..Items.Items import Items, SOTItem
+from ..Items.ItemCollection import ItemCollection
+import math
+
+
+def create_items(world: MultiWorld, location_count: int, options: SotOptionsDerived, itemCollection: ItemCollection,
+                 player: int):
+    update_items_from_options(options, itemCollection)
+
+    # we know the pre fill locations cannot be used
+    location_count = location_count - itemCollection.pre_fill_count
+
+    location_count -= __add_main_items(world, itemCollection, player)
+    location_count -= __add_fill_and_traps(world, itemCollection, player, location_count, options)
+
+    if (location_count != 0):
+        raise Exception(
+            "In item creation, items added to the pool resulted in {} locations remaining.".format(location_count))
+
+
+def update_items_from_options(options: SotOptionsDerived, itemCollection: ItemCollection):
+    itemCollection.informCollectionOfPrefillAction(Items.seal_gh.name, 1)
+    itemCollection.informCollectionOfPrefillAction(Items.seal_ma.name, 1)
+    itemCollection.informCollectionOfPrefillAction(Items.seal_af.name, 1)
+    itemCollection.informCollectionOfPrefillAction(Items.seal_rb.name, 1)
+    itemCollection.informCollectionOfPrefillAction(Items.seal_oos.name, 1)
+    itemCollection.informCollectionOfPrefillAction(Items.pirate_legend.name, 1)
+    itemCollection.informCollectionOfPrefillAction(Items.sail.name, 1)
+
+
+def __add_main_items(world: MultiWorld, itemCollection: ItemCollection, player: int) -> int:
```

Comment:
```suggestion
def __add_main_items(multiworld: MultiWorld, itemCollection: ItemCollection, player: int) -> int:
```
Going to stop commenting on these now, please do not use world: MultiWorld anywhere.
When core reviews this, they _will_ comment on that, so you might as well change it now rather than later.